tests/App/Command/CleanupCommandTest.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace Tests\App\Command;
+
+use App\Command\CleanupCommand;
+use App\Services\CleanupService;
+use App\Storage\RrdStorage;
+use App\Storage\StorageFactory;
+use Prophecy\Argument;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
+use Symfony\Component\Filesystem\Filesystem;
+
+class CleanupCommandTest extends KernelTestCase
+{
+
+    private $logger;
+    private $em;
+    private $fileSystem;
+    private $dirPath;
+    private $application;
+    private $cleanupService;
+    private $storageFactory;
+
+
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    public function setUp()
+    {
+        $kernel = self::bootKernel();
+        $this->dirPath = $kernel->getContainer()->getParameter('kernel.cache_dir'). '/funny';
+        $this->application = new Application($kernel);
+        $this->logger = $this->prophesize("Psr\Log\LoggerInterface");
+        $this->em = $kernel->getContainer()->get('doctrine')->getManager();
+
+        $this->cleanupService = $kernel->getContainer()->get('App\Services\CleanupService');
+
+        $this->storageFactory = $kernel->getContainer()->get('App\Storage\StorageFactory');
+
+        $this->fileSystem = new Filesystem();
+        $this->setupDirectory();
+    }
+
+    public function testExecute()
+    {
+        $cleanUpCommand = new CleanupCommand($this->logger->reveal(), $this->storageFactory, $this->cleanupService);
```

Comment:
Don't use a mocked `$logger`. Let it output to a log file so we can troubleshoot if anything is wrong.